web/client/codechecker_client/cmd_line_client.py

```diff
@@ -325,16 +325,6 @@ def check_run_names(client, check_names):
 
 
 def check_deprecated_arg_usage(args):
-    if 'suppressed' in args:
```

Comment:
Was this suppressed param already a no-op?

Reply:
Yes, it was not used. `--review-status` is used instead.